front/components/home/ContentComponents.tsx

```diff
@@ -196,3 +208,102 @@ export const Strong = ({ children, className = "" }: ContentProps) => (
     {children}
   </strong>
 );
+
+export function CloudConnectorsSection() {
+  return (
+    <div className="rounded-2xl bg-gray-50 px-6 py-8 sm:py-10 md:px-12 md:py-12">
+      <div className="flex flex-col items-center gap-6 sm:gap-8 md:flex-row md:space-x-16">
```

Comment:
We can probably reduce spacing for mobile?

<img width="338" alt="Screenshot 2025-04-11 at 16 13 44" src="https://github.com/user-attachments/assets/990b9200-ee14-4300-815d-5a648deb4bbb" />